### ISSUE TITLE
Update Error Page

### ISSUE THREAD

**Issue by danielchegwidden** [danielchegwidden/datasciencewithdaniel#5]
The error page is what people see when something goes wrong with the site or they try to navigate to a page that does not exist. Currently the error page is a basic HTML file (/source/error.html). This will need to be updated to be more user-friendly and contain more information and be a much nicer screen to land on.

**Comment by daniel-plainview** [comment]
Do you have anything in mind ?

**Comment by danielchegwidden** [comment]
yeah some sort of funny but appropriate image and once we have finalised our marketing materials maybe some of that too and then links back to the main site I think will do. Just a nicer feel to it

**Comment by daniel-plainview** [comment]
I am not a digital artist , however , I do know someone who is and would love to show some ideas

**Comment by danielchegwidden** [comment]
![image](https://user-images.githubusercontent.com/69413854/105437587-4eb3ec80-5c9c-11eb-8b3a-d66bf9618501.jpeg)

This is the 'no notifications' from GitHub which is sort of what I was thinking. Some fun design that people are actually happy to find the error page

**Comment by daniel-plainview** [comment]
We have to make sure that whatever we use in open source or something that is our own

**Comment by daniel-plainview** [comment]
I have sent you a binary file.. a tentative one
